docs/en/sql-reference/functions/hash-functions.md

```diff
@@ -442,7 +442,7 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
 
 ``` text
 ┌─MurmurHash3──────┬─type────────────┐
-│ 6�1�4"S5KT�~~q │ FixedString(16) │
+│ 6�1�4"S5KT�~~q | FixedString(16) │
```

Comment:
This change is incorrect.

Reply:
Can we add hex() function to this example?
```
SELECT hex(murmurHash3_128('example_string')) AS MurmurHash3, toTypeName(MurmurHash3) AS type;
┌─MurmurHash3──────────────────────┬─type───┐
│ 368A1A311CB7342253354B548E7E7E71 │ String │
└──────────────────────────────────┴────────┘
```
It seems more clear to me.

Reply:
Yes, let's add hex.

Reply:
FYI the incorrect change is
`│` to `|`